harbour-foilnotes-zh_cn.ts

```diff
@@ -0,0 +1,319 @@
+        <source>Make sure you don&apos;t forget your password. It&apos;s impossible to either recover it or to access the encrypted notes without knowing it. Better take it seriously.</source>
+        <extracomment>Password confirmation description</extracomment>
+        <translation>请确保你不要忘记你的密码，一旦你忘记它，将无法恢复或访问加密笔记。请留心!</translation>
+    </message>
+    <message id="foilnotes-confirm_password_page-text_field_placeholder-new_password">
+        <source>New password again</source>
+        <extracomment>Placeholder for the password confirmation prompt</extracomment>
+        <translation>再次输入新密码</translation>
+    </message>
+    <message id="foilnotes-confirm_password_page-text_field_label-new_password">
+        <source>New password</source>
+        <extracomment>Label for the password confirmation prompt</extracomment>
+        <translation>新密码</translation>
+    </message>
+    <message id="foilnotes-confirm_password_page-button-confirm">
+        <source>Confirm</source>
+        <extracomment>Button label (confirm password)</extracomment>
+        <translation>确认</translation>
+    </message>
+    <message id="foilnotes-remorse-decrypting">
+        <source>Decrypting</source>
+        <extracomment>Decrypting note in 5 seconds</extracomment>
+        <translation>解密</translation>
+    </message>
+    <message id="foilnotes-remorse-decrypting_selected" numerus="yes">
+        <source>Decrypting %0 selected notes</source>
+        <extracomment>Generic remorse popup text</extracomment>
+        <translation>
+            <numerusform>解密选择的 %0 笔记</numerusform>
+            <numerusform>解密选择的 %0 笔记</numerusform>
+            <numerusform>解密选择的 %0 笔记</numerusform>
+        </translation>
+    </message>
+    <message id="foilnotes-hint-delete_selected">
+        <source>Delete selected notes</source>
+        <extracomment>Hint text</extracomment>
+        <translation>删除选择的笔记</translation>
+    </message>
+    <message id="foilnotes-hint-encrypt_selected">
+        <source>Encrypt selected notes</source>
+        <extracomment>Hint text</extracomment>
+        <translation>加密选择的笔记</translation>
+    </message>
+    <message id="foilnotes-hint-decrypt_selected">
+        <source>Decrypt selected notes</source>
+        <extracomment>Hint text</extracomment>
+        <translation>解密选择的笔记</translation>
+    </message>
+    <message id="foilnotes-foil_pics_warning">
+        <source>Note that Foil Notes and Foil Pics share the encryption key and the password.</source>
+        <extracomment>Warning text, small size label below the password prompt</extracomment>
+        <translation>请注意, "Foil Notes " 和 "Foil Pics" 共享加密密钥和密码。</translation>
+    </message>
+    <message id="foilnotes-hint-swipe_left_to_plaintext">
+        <source>Swipe left to access plaintext notes</source>
+        <extracomment>Left swipe hint text</extracomment>
+        <translation>Там справа незашифрованные записи</translation>
```

Comment:
This one too

Reply:
I have fixed both of them .Please check.

Reply:
But this one still seems to be there.

Reply:
Now，I translate it into  Chinese .Please  check  it  again .